docker/px4-dev/Dockerfile_base

```diff
@@ -47,6 +48,13 @@ RUN apt-get update && apt-get -y --quiet --no-install-recommends install \
 	&& apt-get clean autoclean \
 	&& rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
 
+# gtest
+RUN cd /usr/src/gtest \
+	&& mkdir build && cd build \
+	&& cmake CMakeLists.txt .. && make \
```

Comment:
`cmake CMakeLists.txt ..` -> `cmake ..`

Reply:
https://www.eriksmistad.no/getting-started-with-google-test-on-ubuntu/ yeah actually we don't need the build dir here. @LukasWoodtli

Reply:
I know. But it’s easier to remove unneeded build artefacts. It’s just a matter of removing the build directory after moving the needed build files...

Reply:
Ok so better to apply the suggestion from @dagar

Reply:
All I'm suggesting is dropping the unnecessary `CMakeLists.txt`. When running cmake you just need to point it at the source directory.

Reply:
@dagar Sorry I somehow missed your review...